components/plan-build/bin/hab-plan-build.sh

```diff
@@ -1038,12 +1041,10 @@ _set_build_tdeps_resolved() {
   # dependency could pull in `acme/binutils` for us, as an example. Any
   # duplicate entries are dropped to produce a proper set.
   for dep in "${pkg_build_deps_resolved[@]}"; do
-    tdeps=($(_get_tdeps_for "$dep"))
+    tdeps=$(_get_tdeps_for "$dep")
     for tdep in "${tdeps[@]}"; do
       tdep="$HAB_PKG_PATH/$tdep"
-      pkg_build_tdeps_resolved=(
-        $(_return_or_append_to_set "$tdep" "${pkg_build_tdeps_resolved[@]}")
-      )
+      pkg_build_tdeps_resolved=(_return_or_append_to_set "$tdep" "${pkg_build_tdeps_resolved[@]}")
```

Comment:
As [discussed elsewhere](https://github.com/habitat-sh/habitat/pull/6160#discussion_r258599582), this will need to be changed